packages/integration-tests/tests/api-resources.test.ts

```diff
@@ -0,0 +1,78 @@
+import { Resource } from '@logto/schemas';
+import { managementResource } from '@logto/schemas/lib/seeds';
+
+import { authedAdminApi } from '@/api';
+
+const testApiResource = {
+  id: '',
+  name: 'gallery',
+  indicator: 'https://gallery.logto.io',
+};
+
+describe('admin console api resources', () => {
+  it('should get management api resource details successfully', async () => {
+    const fetchedManagementApiResource = await authedAdminApi
+      .get(`resources/${managementResource.id}`)
+      .json<Resource>();
+
+    expect(fetchedManagementApiResource.id).toBe(managementResource.id);
+    expect(fetchedManagementApiResource.name).toBe(managementResource.name);
+    expect(fetchedManagementApiResource.indicator).toBe(managementResource.indicator);
```

Comment:
deepEqual? or contains?

Reply:
better use `expect.objectContaining` to simplify. for example

<img width="874" alt="image" src="https://user-images.githubusercontent.com/10594507/179959140-5e7f3561-1704-4c1e-89a9-bde2ba9429c7.png">